deckhouse-controller/pkg/controller/module-controllers/release/updater.go

```diff
@@ -211,29 +214,35 @@ func (k *kubeAPI) SaveReleaseData(ctx context.Context, release *v1alpha1.ModuleR
 }
 
 func (k *kubeAPI) updateModuleReleaseDownloadStatistic(ctx context.Context, release *v1alpha1.ModuleRelease,
-	ds *downloader.DownloadStatistic) error {
+	ds *downloader.DownloadStatistic,
+) error {
 	release.Status.Size = ds.Size
 	release.Status.PullDuration = metav1.Duration{Duration: ds.PullDuration}
 
 	return k.client.Status().Update(ctx, release)
 }
 
 type metricsUpdater struct {
-	metricStorage *metric_storage.MetricStorage
+	metricStorage  *metric_storage.MetricStorage
+	enabledModules []string
 }
 
-func newMetricsUpdater(metricStorage *metric_storage.MetricStorage) *metricsUpdater {
+func newMetricsUpdater(metricStorage *metric_storage.MetricStorage, enabledModules []string) *metricsUpdater {
 	return &metricsUpdater{
-		metricStorage: metricStorage,
+		enabledModules: enabledModules,
+		metricStorage:  metricStorage,
 	}
 }
 
 func (m *metricsUpdater) ReleaseBlocked(_, _ string) {
-
 }
 
-func (m *metricsUpdater) WaitingManual(name string, _ float64) {
-	m.metricStorage.GaugeSet("d8_module_release_waiting_manual", 1, map[string]string{"name": name})
+func (m *metricsUpdater) WaitingManual(name string, totalPendingManualReleases float64) {
```

Comment:
`name` argument contains the module release name, not the module name, we can't use it in the following check.

Reply:
would it be difficult to add some tests for metrics? quite annoying to test it manually :\

Reply:
There were too many changes regarding the metrics, I will make a separate pull request for this. In this PR I rolled back the corresponding changes